lib/plugs/group_plug.ex

```diff
@@ -61,6 +62,27 @@ defmodule Castle.Plugs.Group do
   end
   defp set_grouping(conn), do: conn
 
+  defp set_grouping_filters(%{status: nil, assigns: %{group: group}, params: %{"filters" => filters}} = conn) do
+    case parse_filters String.split(filters, ",") do
+      [] -> conn
+      parsed ->
+        assign conn, :group, Map.put(group, :filters, Map.new(parsed))
+    end
+  end
+  defp set_grouping_filters(conn), do: conn
+
+  defp parse_filters([]), do: []
```

Comment:
Minor nit: This could be a good spot for a doc test.

Reply:
Hrmm... unfortunately I made that method private.  But hey, doctests are cool, and I shall endeavor to write some going forward!